build.rs

```diff
@@ -63,7 +63,7 @@ mod generate {
 
         // Then get the names of the format macros
 
-        let fmt_re = Regex::new(r"^\s*#define (?P<full>DRM_FORMAT_(?P<short>[A-Z0-9_]+)) ")?;
+        let fmt_re = Regex::new(r"^\s*#define (?P<full>DRM_FORMAT_(?P<short>[a-zA-Z0-9_]+)) ")?;
```

Comment:
`[a-zA-Z0-9_]` is exactly equal to `\w` :wink:

Reply:
Make a suggestion out of this comment, and I apply it before merging. :)